apps/ledger-live-desktop/src/renderer/components/WebPTXPlayer/index.tsx

```diff
@@ -38,7 +39,21 @@ export const WebViewWrapper = styled.div<WebViewWrapperProps>`
     mobileView.display ? `width: ${mobileView.width ?? 355}px;` : "width: 100%;"}
 `;
 
-export default function WebPTXPlayer({ manifest, inputs }: WebviewProps) {
+export type CustomLoaderType = React.ComponentType<{
+  manifest: LiveAppManifest;
+  isLoading: boolean;
+}>;
+
+export default function WebPTXPlayer({
+  manifest,
+  inputs,
+  CustomLoader,
+}: WebviewProps & {
+  CustomLoader?: React.ComponentType<{
+    manifest: LiveAppManifest;
+    isLoading: boolean;
+  }>;
+}) {
```

Comment:
I think we should be able to directly use the `Loader` props from `WebviewProps` instead of adding a new `CustomLoader` with the same type
```suggestion
  Loader,
}: WebviewProps) {
```